jaxrs/src/test/java/org/killbill/billing/jaxrs/resources/TestJaxRsResourceBase.java

```diff
@@ -17,31 +17,58 @@
 
 package org.killbill.billing.jaxrs.resources;
 
-import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
 
-import org.joda.time.LocalDate;
 import org.killbill.billing.jaxrs.JaxrsTestSuiteNoDB;
-import org.killbill.billing.jaxrs.json.SubscriptionUsageRecordJson.UnitUsageRecordJson;
-import org.killbill.billing.jaxrs.json.SubscriptionUsageRecordJson.UsageRecordJson;
 import org.killbill.billing.payment.api.PluginProperty;
+import org.killbill.billing.util.UUIDs;
+import org.killbill.billing.util.api.CustomFieldApiException;
+import org.killbill.billing.util.api.CustomFieldUserApi;
+import org.killbill.billing.util.collect.Iterables;
+import org.killbill.billing.util.customfield.CustomField;
+import org.mockito.Mockito;
 import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import com.google.common.collect.ImmutableList;
-
 public class TestJaxRsResourceBase extends JaxrsTestSuiteNoDB {
 
-    private final JaxRsResourceBaseTest base = new JaxRsResourceBaseTest();
+    private CustomFieldUserApi customFieldUserApi;
+
+    @BeforeMethod(groups = "fast", alwaysRun = true)
+    public void beforeMethod() {
+        customFieldUserApi = Mockito.mock(CustomFieldUserApi.class);
```

Comment:
Is there a `super` method to call?

Regardless, I think you also need:
```
        if (hasFailed()) {
            return;
        }
```
to work around issues with TestNG where it runs these methods even if the suite is failing.

Reply:
add `if (hasFailed()) { return; }` works.